causal_testing/surrogate/surrogate_search_algorithms.py

```diff
@@ -46,7 +46,7 @@ def fitness_function(ga, solution, idx): # pylint: disable=unused-argument
 
                 ate = surrogate.estimate_ate_calculated(adjustment_dict)
 
-                return contradiction_function(ate)
+                return contradiction_function(ate[0])
```

Comment:
I'm a little worried about the scalability of this as it will only work for float values. If you have categorical data types, this will reduce to a binary treatment between the first and second values in alphabetical order. If you really want to only support float values for this, it would be good to have an explicit check for this to make sure that the user is getting what they expect to get.